lib/rubycritic/cli/options.rb

```diff
@@ -19,6 +19,16 @@ def parse
             @root = path
           end
 
+          opts.on('-b', '--branch BASE_BRANCH,FEATURE_BRANCH', 'Set branches to compare') do |branches|
+            self.base_branch, self.feature_branch = branches.split(',').map { |branch| String(branch) }
```

Comment:
@Onumis This sounds good, also can we merge this with CI mode since we're comparing only the modified files between two branches. So that we can run ``rubycritic --mode-ci`` and set the currently checked-out branch as the "feature branch" and master will be the "base branch"